bootstrap.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+sudo yum -y groupinstall 'Development Tools'
+sudo yum -y install tcl tree vim
```

Comment:
@olivermel Are tree and vim required for the build process?  If not, please remove them.  We only want the minimum needed to build in the bootstrap script.